src/dispatch/incident/views.py

```diff
@@ -304,7 +304,7 @@ def get_incident_forecast(
         if i == 1:
             incidents = create_incident_metric_query(
                 db_session=db_session,
-                filter_spec=common['filter_spec'],
+                filter_spec=common["filter_spec"],
```

Comment:
@kevgliss you may want to double check that your black plugin is running correctly.